app/javascript/components/components/Navigation.js

```diff
@@ -40,13 +40,10 @@ const Navigation = ({
         )}
         {logged_in && (
           <NavItem>
```

Comment:
concerned about this structure

Reply:
I corrected it it seems to be a git hub or a weird error because my code doesn't look like that.